### ISSUE TITLE
Determine what pins we need for I/O

### ISSUE THREAD

**Issue by csinko** [csinko/io-master#27]
Finalize what pins we are going to use

**Comment by codeThatThinks** [comment]
The pin allocation spreadsheet lists all the pins we are going to use.

TLDR:
- ULPI interface used to communicate with USB PHY, these pins cannot be reassigned
- SWD/JTAG header is connected to the default pins for programming
- PORTD is used only for high speed digital outputs to the level shifter (low byte is IO_x_OUT, high byte is IO_x_TRIS)
- PORTE is used only for high speed digital inputs from the level shifter (low byte is NC, high byte is IO_x_IN)
- A dedicated hardware timer is tied to each IO_x_OUT in order to generate output clock signals. Simply enable tristate for the corresponding PORTD pin and disable tristate for the timer output pin.
- I2C1 is used for communication to all level shifter DACs.
- USART1 will be broken out with test points in case it needs to be used for debugging/DFU mode
- DAC_OUT1 is used to control the adjustable regualtor for powering the target device

The pin assignments have been chosen in a way that we could downgrade to the LQFP100 package without any issues to save board space.

**Comment by codeThatThinks** [comment]
@WhoDaWhatNow I've changed some of the pin assignments in order to make the board layout easier. Please review the spreadsheet on drive.
